### DISCUSSION TITLE
Ddt4all log recorder

### DISCUSSION THREAD

**Discussion by monalos** [cedricp/ddt4all#1051] [Q&A]
Hello, how can I save logs when using ddt4all? Where can I see the transactions made?

**Comment by Furtif** [comment]
It would be nice to have dumps with python pickle.load() / pickle.dump() but that would need @shrlnm help if he wants to implement this

**Reply by shrlnm** [reply]
I mentioned ordinar elm logs in Discussion https://github.com/cedricp/ddt4all/discussions/1049

The ddt4all writes elm logs every time uncoditionaly 

options.py

	log = "ddt"

elm.py

                self.lf = open("./logs/elm_" + options.log + ".txt", "at", encoding="utf-8")
                self.vf = open("./logs/ecu_" + options.log + ".txt", "at", encoding="utf-8")

**Reply by Furtif** [reply]
yep but i talk about dumps mode feature
